CodeGeneration/Sources/SyntaxSupport/Child.swift

```diff
@@ -147,8 +147,8 @@ public class Child {
 
   /// If a classification is passed, it specifies the color identifiers in
   /// that subtree should inherit for syntax coloring. Must be a member of
-  /// SyntaxClassification in SyntaxClassifier.h.gyb
-  /// If force_classification is also set to true, all child nodes (not only
+  /// `SyntaxClassification` in `SyntaxClassifier.h.gyb`
```

Comment:
If we use double ticks we can link to `Syntaxassification`.

Reply:
Good point! Changed to double ticks in 482219e.